src/main/java/com/commercetools/sync/producttypes/utils/ProductTypeUpdateAttributeDefinitionActionUtils.java

```diff
@@ -125,7 +126,7 @@
 
             return updateActions;
 
-        } catch (final DuplicateNameException | DuplicateKeyException | DifferentTypeException exception) {
+        } catch (final DuplicateNameException | DuplicateKeyException exception) {
             throw new BuildUpdateActionException(exception);
```

Comment:
Are there unit tests for the case where the `BuildUpdateActionException` is thrown here. I tried to find unit tests for `ProductTypeUpdateAttributeDefinitionActionUtils#buildAttributeDefinitionsUpdateActions` but didn't find any :(

Reply:
Yes, there is https://github.com/commercetools/commercetools-sync-java/blob/master/src/test/java/com/commercetools/sync/producttypes/utils/producttuypeactionutils/BuildAttributeDefinitionUpdateActionsTest.java#L159